### ISSUE TITLE
Binary download link dead

### ISSUE THREAD

**Issue by dantebarba** [dreamstalker/rehlds#756]
Unfortunately I can't download binaries from http://nexus.rehlds.org/nexus/content/repositories/rehlds-dev/rehlds/rehlds/. At this time it's giving me HTTP 500.

Would be possible to mirror the releases or include them in github?

**Comment by Infamous2017** [comment]
And why you dont download it here http://teamcity.rehlds.org/overview.html ?

**Comment by andrewlindberg** [comment]
You cand find it here https://github.com/dreamstalker/rehlds/releases

**Comment by StevenKal** [comment]
Same issue as there: https://github.com/s1lentq/ReGameDLL_CS/issues/515

As I said, priority should be the latest, but it would be nice if devs could make some efforts at make available again all of those archives from that link (which was easy to get for us), at least in a secondary "archives" link below the main ones, if in case we quickly need an old build for testing & compatibility purpose, etc.
Or explain us what was the problem with the old listing way and why you changed it (my first guessing is you want to enforce the latest release, which is preferable, but puting the archives link in a "not much visible link" will be also welcomed).

**Comment by Darko** [comment]
> And why you dont download it here http://teamcity.rehlds.org/overview.html ? Login as Guest and good.

How do you even download it from here?

**Comment by StevenKal** [comment]
After login as guest, I personnaly tried something like:
ReHLDS -> Changelog tab -> According to the changelog listing, search for the files (on the right) to a version you may wish -> Click on the "x file(s)" link -> Select "Builds" -> Publish (Windows + Linux) -> Artifacts (that a weird name which make me think to Tomb Raider's artefacts! Can not just be named "file"?).
But the system point to the latest versions so... it seems we can not get an older build, and devs not O.K. to let us doing it. So be it.
And even if there is a possible way to get an older build, this is probably a pain via this teamcity website.

**Comment by dantebarba** [comment]
> And why you dont download it here http://teamcity.rehlds.org/overview.html ? Login as Guest and good.

Yes I finally stumbled upon that source and I could download it successfully. But I was just pointing out that the link in the README.MD was not working anymore.

**Comment by Infamous2017** [comment]
> 
> 
> > And why you dont download it here http://teamcity.rehlds.org/overview.html ? Login as Guest and good.
> 
> How do you even download it from here?

https://www.youtube.com/watch?v=MWZjWr_s7RA

**Comment by AdrianJagodznski** [comment]
All download links dead, how can i download now?

**Comment by leki85** [comment]
![Screenshot from 2020-04-06 11-44-49](https://user-images.githubusercontent.com/15772641/78545573-49181380-77fc-11ea-9aff-2ec440e0cde3.png)
Seems domain is not paid. :(

**Comment by andrewlindberg** [comment]
Here https://github.com/dreamstalker/rehlds/releases

**Comment by leki85** [comment]
10x man. I don't know how i didn't see this. :)

**Comment by Infamous2017** [comment]
https://github.com/dreamstalker/rehlds/releases  click at assets and click .zip file .  whats so hard there for u ?   the old Site is offline .   Idk why or idk maybe it comes back we will see.

**Comment by Infamous2017** [comment]
There are still the master versions here. It's a shame that there are no non-master versions, because I only use them because they fix bugs faster. You have to register on the site.

https://dev-cs.ru/resources/64/history   (REHLDS)
https://dev-cs.ru/resources/67/history  (REGAME)
https://dev-cs.ru/resources/73/history  (REAPI)
https://dev-cs.ru/resources/208/history (METAMOD-R)

**Comment by WPMGPRoSToTeMa** [comment]
I've found the old ip address of the domain (see [here](https://securitytrails.com/domain/rehlds.org/history/a)). You just need to add these lines to `hosts` file:
```
46.105.47.212 rehlds.org
46.105.47.212 nexus.rehlds.org
46.105.47.212 teamcity.rehlds.org
```

**Comment by Infamous2017** [comment]
WPMGPRoSToTeMa   THX MAN this works like u said.   Good Job.

**Comment by WPMGPRoSToTeMa** [comment]
As I can see the domains are assigned to the old ip address (since April 20th), so editing the `hosts` file is not needed anymore.

**Comment by Infamous2017** [comment]
Thx for the Info.
